tsconfig.json

```diff
@@ -13,7 +13,8 @@
         "noImplicitAny": true,
         "noImplicitReturns": true,
         "noUnusedLocals": true,
-        "allowSyntheticDefaultImports": true
+        "allowSyntheticDefaultImports": true,
+        "esModuleInterop": true
```

Comment:
Changing tsconfig seems more sensitive and more likely to not be accepted upstream, what do we gain from this configuration?